tests/target/chains-no-overflow.rs

```diff
@@ -9,20 +9,16 @@ fn main() {
                        .ddddddddddddddddddddddddddd
                        .eeeeeeee();
 
-    x().y(|| {
-           match cond() {
-               true => (),
-               false => (),
-           }
+    x().y(|| match cond() {
```

Comment:
I'm not keen on introducing unnecessary parens - the compiler even lints against it in `if` expressions, etc. The only reason to have the braces is to make a multi-line expression more coherent, and I don't think parens accomplish that as well.

I'm not super-keen on this change either, but I have grown more used to it and I like it more than any of the other alternatives.